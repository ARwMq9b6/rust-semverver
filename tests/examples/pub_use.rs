pub mod old {
    pub mod a {
        pub struct Abc;
    }

    pub use old::a as b;
}

pub mod new {
    pub mod a {
        pub struct Abc;
    }

    pub mod b {
        pub use new::a::*;
    }
}
