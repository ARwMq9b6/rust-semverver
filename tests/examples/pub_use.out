version bump: 1.0.0 -> (Patch) -> 1.0.1
